app/router.js

```diff
@@ -1,22 +1,18 @@
 import EmberRouter from '@ember/routing/router';
 import config from './config/environment';
-import { inject as service } from '@ember/service';
 
 const Router = EmberRouter.extend({
-  navbar: service(),
   location: config.locationType,
-  rootURL: config.rootURL,
```

Comment:
So it looks like you're removing some stuff as a result of the rebases and merge conflicts, these need to be here 👍